### PR TITLE
fix trendmicro deep security

### PR DESCRIPTION
**Pull request by PierrickV**
- fix(Trend Micro/DeepVisibility): Wrongly called `CEF` fields.
- feat(Trend Micro/DeepVisibility): Improve basic smart description.

### CODE REVIEW

File: Trend Micro/deep-security/ingest/parser.yml [hunk 1]
```diff
@@ -75,7 +75,7 @@ stages:
           "25": "Add/Remove Programs"
           "26": "Services"
         mapping:
-          parsed_event.message.cs4: trendmicro.ResourceType
+          parsed_event.message.get("Resource Type"): trendmicro.ResourceType
```
Comment (squioc) [inline Trend Micro/deep-security/ingest/parser.yml:78]: I'm not sure this notation will work.
I think the CEF stage should be fixed to replace spaces with an underscore.

File: Trend Micro/deep-security/ingest/parser.yml [hunk 2]
```diff
@@ -88,7 +88,7 @@ stages:
           "75": "High"
           "100": "Very High"
         mapping:
-          parsed_event.message.cs5: event.risk_score_norm
+          parsed_event.message.get("Risk Level"): event.risk_score_norm
```
Comment (squioc) [inline Trend Micro/deep-security/ingest/parser.yml:91]: Same as above

**Review by squioc — CHANGES_REQUESTED** [review]

File: Trend Micro/deep-security/ingest/parser.yml (hunk 1 above)
Comment (PierrickV) [reply Trend Micro/deep-security/ingest/parser.yml]: Hello @squioc, please review my latest commit where I tried to use an intermediate field to bypass this limitation. 

I will try to improve this work once I have more events.

File: Trend Micro/deep-security/ingest/parser.yml [hunk 3]
```diff
@@ -42,10 +42,12 @@ stages:
           trendmicro.FragmentationBits: '{{parsed_event.message.get("Fragmentation Bits")}}'
           trendmicro.TCPFlags: '{{parsed_event.message.get("TCP Flags").split(" ")}}'
           trendmicro.TrendMicroDsRelevantDetectionNames: '{{parsed_event.message.TrendMicroDsRelevantDetectionNames.split(";")}}'
+          trendmicro.ResourceTypeId: '{{parsed_event.message.get("Resource Type")}}'
           container.name:  '{{parsed_event.message.get("Container")}}'
           file.name: '{{parsed_event.message.get("Infected Resource")}}'
           host.id: '{{parsed_event.message.get("Host ID")}}'
           event.reason: '{{parsed_event.message.get("actionReason")}}'
+          event.severity: 'parsed_event.message.get("Risk Level")'
```
Comment (squioc) [inline Trend Micro/deep-security/ingest/parser.yml:50]: The jinja brace are missing.
To respect the ECS specification, [`event.severity`](https://www.elastic.co/guide/en/ecs/8.4/ecs-event.html#field-event-severity) should be a long
```suggestion
          event.severity: '{{parsed_event.message.get("Risk Level") | int}}'
```

File: Trend Micro/deep-security/ingest/parser.yml [hunk 4]
```diff
@@ -88,7 +90,7 @@ stages:
           "75": "High"
           "100": "Very High"
         mapping:
-          parsed_event.message.cs5: event.risk_score_norm
+          event.severity: event.risk_score_norm
```
Comment (squioc) [inline Trend Micro/deep-security/ingest/parser.yml:93]: I wonder if you don't have to change the mapping with integer on the left side.

**Review by squioc — CHANGES_REQUESTED** [review]

File: Trend Micro/deep-security/ingest/parser.yml (hunk 3 above)
Comment (PierrickV) [reply Trend Micro/deep-security/ingest/parser.yml]: Good catch! thank you

File: Trend Micro/deep-security/ingest/parser.yml (hunk 4 above)
Comment (PierrickV) [reply Trend Micro/deep-security/ingest/parser.yml]: Applied ! =)

Comment (PierrickV) [reply Trend Micro/deep-security/ingest/parser.yml]: I used the field `risk_score_norm` which seems more appropriate

**Review by squioc — APPROVED** [review]

**Pull request merged**